setup.py

```diff
@@ -65,9 +70,9 @@ def _load_py_module(fname, pkg="flash"):
     }
 )
 # some extra combinations
-extras_req["vision"] = extras_req["image"] + extras_req["video"]
-extras_req["all"] = extras_req["vision"] + extras_req["tabular"] + extras_req["text"]
-extras_req["dev"] = extras_req["all"] + extras_req["test"] + extras_req["docs"]
+extras_req["vision"] = _expand_reqs(extras_req, ["image", "video"])
+extras_req["all"] = _expand_reqs(extras_req, ["vision", "tabular", "text", "audio", "graph"])
```

Comment:
Graph has some funny dependencies so I think we will have to remove it from all
```suggestion
extras_req["all"] = _expand_reqs(extras_req, ["vision", "tabular", "text", "audio"])
```